data/basics/userBalance_test.go

```diff
@@ -131,102 +131,6 @@ func getSampleAccountData() AccountData {
 	}
 }
 
-func TestEncodedAccountDataSize(t *testing.T) {
```

Comment:
partially re-used in a similar test in accountdb_test for base account data and resource data max size checks